requirements/multi-modal.txt

```diff
@@ -26,4 +26,5 @@ torchvision
 transformers>=4.27.1
 # triton==2.0.0.dev20221120
 unicodedata2
+xformers>=0.0.17
```

Comment:
remove xformer from multi-modal as a necessary one

Reply:
> remove xformer from multi-modal as a necessary one

Okay, it's resolved